Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -1794,6 +1803,11 @@ open class NavigationMapView: UIView {
                    layerInfo.type.rawValue != "symbol",
```

Comment:
Nesting the `if` statement below gets confusing. I think this would more clearly communicate the intention:

```swift
(layerInfo.type.rawValue != "symbol" and not (layerInfo.type.rawValue == "circle" and mapView.mapboxMap.style.isPersistentLayer(id: layerInfo.id)))
```